### PR TITLE
chore: updated signing disclosure for documenso release.com

### PR DESCRIPTION
**Pull request by Lbostach**
---
name: Pull Request
about: Submit changes to the project for review and inclusion
---

## Changes Made

- Change 1
- Change 2
- ...

## Testing Performed

- Tested feature X in scenario Y.
- Ran unit tests for component Z.
- Tested on browsers A, B, and C.
- ...

## Checklist

- [ ] I have tested these changes locally and they work as expected.
- [ ] I have added/updated tests that prove the effectiveness of these changes.
- [ ] I have updated the documentation to reflect these changes, if applicable.
- [ ] I have followed the project's coding style guidelines.
- [ ] I have addressed the code review feedback from the previous submission, if applicable.

### CODE REVIEW

**Comment by Lbostach** [comment]
@jasonong

File: docker/development/compose.yml [hunk 1]
```diff
@@ -12,8 +12,8 @@ services:
       - POSTGRES_DB=documenso
     ports:
       - 54320:5432
-#    networks:
- #      - my-custom-created-network
+    # networks:
```
Comment (jasonong) [inline docker/development/compose.yml:15]: Why are there changes for networks?

**Review by jasonong — APPROVED** [review]
@Lbostach A quick question regarding changes to indentations for network in docker compose file

File: docker/development/compose.yml (hunk 1 above)
Comment (Lbostach) [reply docker/development/compose.yml:15]: Oversight on my part, will correct indentation before pushing to uat.

**Pull request merged**